src/cultionet/scripts/config.yml

```diff
@@ -2,12 +2,13 @@ image_vis:
   - evi2
   - gcvi
   - kndvi
+  - green
 
-# The regions to process (start, end)
 regions:
   - 1
   - 1
 
+region_id_file:
```

Comment:
@jgrss , what do you think if we add region_id_file here in the config file instead of adding additional argument to the CLI?

Reply:
@nnguyen622 that could work with a default of `!!null`. And then you'll add a check for both of those options?

Reply:
hi @jgrss, good point. I will add in check for both of these options. 👍